src/list-seqs.py

```diff
@@ -1,17 +1,27 @@
-import argparse
+import sys
 
+def list_seqs():
 
-def main():
-    argparser = argparse.ArgumentParser(
-        description="Extract the sequences from a simple-fastq file")
-    argparser.add_argument(
-        "fastq",
-        type=argparse.FileType('r')
-    )
-    args = argparser.parse_args()
+    inFile = sys.argv[1]  
+    with open(inFile,'r') as j:
+        lines = j.readlines()
 
-    print(f"Now I need to process the records in {args.fastq}")
+    record_list = []
+    header = ''
+    sequence = ''
+    for line in lines:
+        line = line.strip()
+        if line.startswith('@'):
+            if header != "":
+                record_list.append([header, sequence])
+                sequence = ""
+            header = line[1:]
+        else:
+            sequence = sequence + line
```

Comment:
Unless they have changed that very recently in Python, `x = x + y` for strings will copy all of `x` and all of `y` into a new string. Thus, if you do this in a loop, you get a quadratic time algorithm and not a linear one. That can kill the performance of your tools very quickly.

Currently, using `x += y` will work as an amortised linear concatenation, so you can do that instead. Alternatively, an something that the standard guarantee, not just the current implementation, you can collect the strings you want to concatenate in a list, `w`, and get your concatenated string with `''.join(w)`.